GiniVisionExample/GiniVisionExampleUnitTests/PDFNoResultsViewControllerTests.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  PDFNoResultsViewControllerTests.swift
+//  GiniVisionExampleUnitTests
+//
+//  Created by Enrique del Pozo Gómez on 11/24/17.
+//  Copyright © 2017 Gini. All rights reserved.
+//
+
+import XCTest
+@testable import GiniVisionExample
+
+final class PDFNoResultsViewControllerTests: XCTestCase {
```

Comment:
This file is so empty and alone...

Reply:
I know, but I didn't want to remove it since I will probably add some tests when I have the final design